package.json

```diff
@@ -68,5 +67,8 @@
     "typechain": "^8.1.1",
     "typescript": "^4.9.3"
   },
-  "packageManager": "yarn@3.3.0"
+  "packageManager": "yarn@3.3.0",
+  "dependencies": {
+    "@big-whale-labs/constants": "^0.1.73"
+  }
```

Comment:
It should be above `devDependencies`

Reply:
Also, do we need to move in from `devDependencies` to `dependencies`?

Reply:
We probably can have it anywhere we want